compiler/src/dotty/tools/dotc/typer/Inliner.scala

```diff
@@ -1235,9 +1235,8 @@ class Inliner(call: tpd.Tree, rhsToInline: tpd.Tree)(using Context) {
           else Block(cond1 :: Nil, selected)
         case cond1 =>
           if (tree.isInline)
-            errorTree(tree, em"""cannot reduce inline if
-                                | its condition   ${tree.cond}
-                                | is not a constant value""")
+            errorTree(tree, em"""Cannot reduce `inline if` with non constant value.
```

Comment:
I think the old error message is clearer.

Reply:
The issue with the previous message is that the condition is printed in the middle of the message. That condition may be long or have several lines. Ideally we should have a message where the tree is printed last.

Reply:
```
Cannot reduce `inline if` because its condition is not a constant value:
$cond1
```
I keep meaning to see that Ralph Fiennes movie, "The Constant Gardener."

If we could use that title for a narrative about compilation, then we'd have the same problem as that other Fiennes movie, "The English Patient," whether to emphasize the adjective or the noun.

The suggested edit avoids "non constant" --> "non-constant" --> "inconstant" discussion.

Reply:
I used ```"Cannot reduce `inline if` because its condition is not a constant value: $cond"```.

Reply:
OK, I see the reasoning now.